lib/url.js

```diff
@@ -563,7 +563,7 @@ Url.prototype.format = function() {
   }
 
   var protocol = this.protocol || '';
-  var pathname = this.pathname || '';
+  var pathname = (this.pathname + '') || '';
```

Comment:
Please don't change `lib/url.js` behavior unless you need to. And if you need to, add a test.